baldrick/plugins/github_pull_requests.py

```diff
@@ -99,7 +99,10 @@ def process_pull_request(repository, number, installation, is_new=False):
     for label in pr_handler.labels:
         if label in skip_labels:
             if skip_fails:
-                pr_handler.set_status('failure', "Skipping checks due to {0} label".format(label), current_app.bot_username)
+                pr_handler.set_check(
```

Comment:
Do we need this in check land?

(I honestly don't know, but do we have the same issue with having to override failed statues with checks?)

Reply:
Currently, the real world usage at `astropy` is something like "skipping checks due to Experimental label" with ❌ , which is a clear reminder that we need to revisit it when it is no longer experimental. But if it just does not show up, then what does that mean?

Reply:
I vote keeping this for now and push to testing stage and see what happens.